tools/check-bash-scripts

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+path=$1
+shfmt -d "${path}"
```

Comment:
In accordance with my other suggestion:

```suggestion
executable=$1 path=$2
"${executable:-shfmt}" -d "${path}"
```

Reply:
@Martchus why do we need this wrapper at all?

Reply:
the `-d` option has two effects: 1. it shows the diff 2. the exit code will be non-zero if there is a diff
The directory parameter is always needed.